drivers/platform/xilinx/spi_extra.h

```diff
@@ -98,4 +98,24 @@ typedef struct xil_spi_desc {
 	void			*instance;
 } xil_spi_desc;
 
-#endif // SPI_EXTRA_H_
\ No newline at end of file
+/**
+ * @brief Xilinx platform specific SPI platform ops structure
+ */
+extern const struct spi_platform_ops xil_platform_ops;
```

Comment:
I think we can do this to avoid the stuff with extern which maybe is a little messy:
```suggestion
#define SPI_PLATFORM_OPS (&(struct spi_platform_ops) {\
               .spi_ops_init = xil_spi_init,\
               .spi_ops_write_and_read = xil_spi_write_and_read,\
               .spi_ops_remove = xil_spi_remove\
}))
```

Reply:
And in spi_init_param will be directly:
`.platform_ops = SPI_PLATFORM_OPS`

Reply:
I think this is a matter of preferences.

Reply:
ok, this way it would be in just one place, but I don't really want this change.  It was just a suggestion.